en/developer_guide/logs.rst

```diff
@@ -0,0 +1,97 @@
+Logs
+====
+
+Our backoffice logging is based on `Monolog`_.
+Open Orchestra provides an interface to display all the user actions on the website.
+
+.. image:: ../../images/log_preview.png
+
+Configuration
+-------------
+
+Monolog uses channels to record the log. To link a channel and a record, you will have to define an handler.
+The configuration of the handler can be placed in the ``app/config/log.yml`` file.
+
+For instance, if you want to store the logs in mongo, you will have to define a handler with the database connection information.
+The level entry indicates the information type to record in the logs.
+Each level is a number which value is standardized in the PSR-3 recommendation.
+For example 200 it's for info. The different level are defined in the `Monolog documentaion`_
+
+.. code-block:: yaml
+
+        mongo:
+            type: mongo
+            level: 200
+            mongo:
+                host: "localhost"
+                database: "%open_orchestra_cms.mongodb.database%"
+                collection: log
+            channels: [openorchestra]
+
+
+The channels key defines the channels where are written the logs.
```

Comment:
``where the log are written``.

Instead of written, I would say published, because the log are only published, then there are handler to listen to a channel and write them to the database or file